crates/re_viewer/src/app.rs

```diff
@@ -1058,12 +1056,12 @@ impl eframe::App for App {
             self.command_sender.send_ui(cmd);
         }
 
-        self.run_pending_ui_commands(frame, egui_ctx, &app_blueprint, store_context.as_ref());
+        self.handle_dropping_files(egui_ctx);
 
+        // Run pending commands last:
```

Comment:
Comment explaining why this is important would be nice if we want to keep things ordered this way.